composer.json

```diff
@@ -20,7 +20,9 @@
     },
     "require-dev": {
         "eloquent/phony-phpunit": "^3.0",
-        "phpunit/phpunit": "^6.0"
+        "league/container": "^2.0",
```

Comment:
Why?

Reply:
I like testing with real implementations (to show real example usage). I can change to use a mock / stub if you want.

Reply:
So... okay to keep? Not 100% sure how to interpret your :+1:

Reply:
No, remove it please.

Reply:
Okay.